www/%username/widgets/index.html.spt

```diff
@@ -28,22 +28,26 @@ profile_url = participant.url()
     <a href="{{ profile_url }}donate"><img src="/assets/widgets/donate.svg"></a>
 
     <h3>{{ _("Giving & Receiving Widgets") }}</h3>
+    % if participant.hide_receiving and participant.hide_giving
+        <p>{{ _("To enable widgets, uncheck 'hide' giving and/or receiving under Privacy in <a href=\"{0}\">your settings</a>."|safe, profile_url) }}</p>
+    % endif
+    % if not participant.hide_receiving
+        <p>{{ _("Use this code to add to your website a widget which displays the amount of donations you're receiving on Liberapay:") }}</p>
 
-    <p>{{ _("Use this code to add to your website a widget which displays the amount of donations you're receiving on Liberapay:") }}</p>
+        <pre>&lt;script src="{{ profile_url }}widgets/receiving.js"&gt;&lt;/script&gt;</pre>
 
-    <pre>&lt;script src="{{ profile_url }}widgets/receiving.js"&gt;&lt;/script&gt;</pre>
+        <p>{{ _("Here's what it looks like:") }}</p>
 
-    <p>{{ _("Here's what it looks like:") }}</p>
-
-    <p><script src="receiving.js"></script></p>
-
-    <p>{{ _("Or, if you'd like a widget that shows how much you're giving:") }}</p>
+        <p><script src="receiving.js"></script></p>
+    % endif
+    % if not participant.hide_giving
+    <p>{{ _("Use this code to add to your website a widget which displays the amount of you're giving on Liberapay:") }}</p>
```

Comment:
The new indentation here isn't consistent with the one below, and it's created a diff that's difficult to read.